go/def.bzl

```diff
@@ -206,7 +206,8 @@ def _is_external(p):
   """Checks if the string starts with ../"""
   return p[0:3] == '../'
 
-def emit_go_compile_action(ctx, sources, deps, out_lib, extra_objects=[]):
+def _emit_go_compile_action(ctx, sources, deps, out_lib,
+                           extra_objects=[], gc_goopts=[]):
```

Comment:
I'm not sure if Skylark is affected by this. I figured it's better to just make the arguments explicit though.

Will submit when CI is done again (Mac build is taking forever to run these days).